Managers/MapManager.gd

```diff
@@ -214,7 +234,7 @@ func _ready() -> void:
 					events[event] += 1
 					total_nb_rooms += 1
 			for k: String in events:
-				print("Event " + k + " has " + str(events[k]) + " rooms. (is " + str(float(events[k]) * 100 / total_nb_rooms).pad_decimals(2) + "%, expected: " + str(expected_probabilities[k]) + "%)")
+				print("Event " + k + " has " + str(events[k]) + " rooms (expected: "+ str(float(expected_probabilities[k] * 61.0/100)).pad_decimals(2) +"). (The percentage is " + str(float(events[k]) * 100 / total_nb_rooms).pad_decimals(2) + "%, expected: " + str(expected_probabilities[k]) + "%)")
```

Comment:
ah i was also wondering, why multiply by `61.0/100` ?

Reply:
the probability is not between 0 and 1 but 0 and 100, so it still needs to be divided by 100 to get the number of rooms we want

Reply:
yes, for the divide by 100, but why multiply by 61 ?

Reply:
oh right it was just hard coded, I will change it